app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Qt Markdown editor"
+DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML preview, math expressions, code and markdown syntax highlighting."
```

Comment:
Please wrap this to 80-column limit as in the [Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines)